department-of-veterans-affairs/GIBCT-Data-Service
# Added Preferred Provider Column to Weams

## Description 
As a GIDS Tool user, I need the ability to upload a WEAMS.csv file containing a preferred provider indicator so that Comparison Tool users can identify preferred VET TEC providers.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19179

## Testing done

1. Run `bundle exec rake db:migrate`
2. Ensure the stem indicator column is present in the database
3. Run `bundle exec rails s -p 4000`
4. Open the GI Bill Comparison Tool Dashboard
5. Export Weam.csv and ensure preferred_provider column is generated. 
6. Logged into the GIDS dashboard and exported a copy of the Weam CSV to ensure the preferred column was present. 
7. Logged into the GIDS dashboard and exported a copy of the Institutions CSV to ensure the preferred column was present. 

## Screenshots
![Screen Shot 2019-07-10 at 9 21 05 AM](https://user-images.githubusercontent.com/50601724/60972360-205a0080-a2f4-11e9-9f5e-230ea4095a3b.png)


## Acceptance criteria
- [X] A WEAMS.CSV file containing a "Preferred Provider" column can be uploaded into GIDS.

## Definition of done
